website/pages/en/index.js

```diff
@@ -218,9 +215,9 @@ class Index extends React.Component {
             title: '[Puppeteer Testing Library](./pptr)',
           },
           {
-            image: `${baseUrl}img/construction-128x128.png`,
+            image: `${baseUrl}img/react-128x128.png`,
             imageAlign: 'top',
-            title: '[And more...](./docs/ecosystem-user-event)',
+            title: '[Native Testing Library](./react-native)',
```

Comment:
Can you leave the ecosystem link here and make a standard thumbnail for react native?

Reply:
Yep I can leave the ecosystem one in there. 

Disclosure, the reason I took it out was that it was the last non-implementation package so it seemed out of place, and it looks maaad weird when there's two items in this last row. Do you think there's one more thing we could put in this table so it's all 3 across?

Also react native and react have the same logo weirdly. Do you want me to just copy the react logo and name it react-native?

Reply:
You can change it to 2, 3, or 4 columns, might look better

Reply:
Looks like the RN site uses a white-on-black React logo, how about using a black one for RN?

Reply:
I'll try to do this this evening, as part of it do you want me to remove the link to each landing page? I think we should probably delete the pages separately because some of the libraries (at least RTL) link to their landing page from their readme.

Reply:
You can place redirects in netlify.toml from the landing pages to the docs pages